src/main/java/com/limechain/network/ProtocolUtils.java

```diff
@@ -31,16 +19,8 @@ public static String getKadProtocol(String chainId) {
         return String.format("/%s/kad", chainId);
     }
 
-    public static String getGrandpaProtocol(String chainId) {
-        return String.format("/%s/grandpa/1", grandpaProtocolChain(chainId));
-    }
-
-    //TODO: figure out a more elegant solution
-    private static String grandpaProtocolChain(String chainId) {
-        return chainId.equals("dot") ? "paritytech" : chainId;
+    public static String getGrandpaProtocol() {
+        return "/paritytech/grandpa/1";
```

Comment:
Are we sticking to the legacy protocol string?

Reply:
Until we add configuration for the host, yes